ouroboros-network/src/Ouroboros/Network/Diffusion/P2P.hs

```diff
@@ -1339,9 +1339,6 @@ isFatal ResourceExhausted = True
         -- ENFILE            -- socket, accept
         -- ENOBUFS           -- socket, accept
         -- ENOMEM            -- socket, accept
-isFatal UnsupportedOperation = True
-        -- EADDRNOTAVAIL     -- connect
-        -- EAFNOSUPPRT       -- connect
```

Comment:
Could you add a haddock comment why `EAFNOSUPPORT` should not be included.

Reply:
Done!